examples/basic/main.tf

```diff
@@ -9,10 +9,11 @@ resource "random_pet" "this" {
 module "dynamodb_table" {
   source = "../../"
 
-  name        = "my-table-${random_pet.this.id}"
-  hash_key    = "id"
-  range_key   = "title"
-  table_class = "STANDARD"
+  name                        = "my-table-${random_pet.this.id}"
+  hash_key                    = "id"
+  range_key                   = "title"
+  table_class                 = "STANDARD"
+  deletion_protection_enabled = true
```

Comment:
lets just demonstrate setting to `false` since the examples are intended to be deployed and destroyed (which this would prevent that)